doc/learn_more.rst

```diff
@@ -16,10 +16,13 @@
 
 .. You should have received a copy of the GNU General Public License
  along with OpenShot Library.  If not, see <http://www.gnu.org/licenses/>.
+.. _learn_more_ref:
 
 Learn More
 ==========
-We are working hard to expand this user guide, but if you are stuck and don't know where to turn,
-please submit a question or bug report here: Issues_.
+We are working hard to expand this user guide and to improve OpenShot Video Editor, but if you are stuck and don't know where to turn, OpenShot has several sources for additional information.
 
-.. _Issues: https://github.com/OpenShot/openshot-qt/issues/
+ #. OpenShot has several `YouTube videos <https://www.youtube.com/results?search_query=openshot+video+editor+jonathan+thomas>`_ available to help you see how to use the application.
```

Comment:
Let's use our official playlist for tutorials: https://www.youtube.com/watch?v=1k-ISfd-YBE&list=PLymupH2aoNQNezYzv2lhSwvoyZgLp1Q0T&index=1